src/flightlog_fielddefs.js

```diff
@@ -526,6 +526,10 @@ export function adjustFieldDefsList(firmwareType, firmwareVersion) {
     if (semver.lt(firmwareVersion, "4.6.0")) {
       DEBUG_MODE.splice(DEBUG_MODE.indexOf("D_MAX"), 1, "D_MIN");
     }
+    if (semver.gte(firmwareVersion, "4.6.0")) {
+      DEBUG_MODE.splice(DEBUG_MODE.indexOf('GPS_RESCUE_THROTTLE_PID'), 1, 'AUTOPILOT_ALTITUDE');
+      DEBUG_MODE.splice(DEBUG_MODE.indexOf("GYRO_SCALED"), 1);
```

Comment:
maybe skip the gyro_scaled for now, merge only the autopilot_altitude :man_shrugging:  ??
```suggestion
```

do gyro_scaled in another PR and wait?

Reply:
Firmware PR is ready to merge, just waiting for client support to be added first :upside_down_face: